invenio_vocabularies/contrib/funders/schema.py

```diff
@@ -8,9 +8,16 @@
 
 """Funders schema."""
 
+from functools import partial
+
 from flask_babelex import lazy_gettext as _
+from invenio_records_resources.services.records.schema import BaseRecordSchema
 from marshmallow import Schema, fields, validate
-from marshmallow_utils.fields import SanitizedUnicode
+from marshmallow_utils.fields import IdentifierSet, SanitizedUnicode
+from marshmallow_utils.schemas import IdentifierSchema
+
+from ...services.schema import BaseVocabularySchema
+from .config import funder_schemes
 
 
 class FunderRelationSchema(Schema):
```

Comment:
Major: This field is missing an `id` field. Otherwise, the "relation" won't be saved. The way the relation schema work is:

For example, the user wants to add 2 funders to it's record. One exists in our system with id `funder-id-1`, the other does not exist and its name is `JDoe funding agency`. The record would be sent (and stored in DB) as:

```json
{ ...
    "funders": [
        {"id": "funder-id-1"},
        {"name": "JDoe funding agency"}
    ]
}
```

Then, internally the record would be denormalized. Therefore the result (for the end user) would be:

```json
{ ...
    "funders": [
        {"id": "funder-id-1", "name": ..., "title":..., "scheme"...},
        {"name": "JDoe funding agency"}
    ]
}
```